@xen-orchestra/lite/src/views/host/HostNetworkView.vue

```diff
@@ -48,13 +50,15 @@ const selectedPif = useRouteQuery<XenApiPif | undefined>('id', {
 
 <style lang="postcss" scoped>
 .host-network-view {
-  display: grid;
-  grid-template-columns: minmax(0, 1fr) 40rem;
-
   .container {
     height: fit-content;
-    gap: 4rem;
     margin: 0.8rem;
+    gap: 4rem;
+  }
+
+  @media (min-width: 1024px) {
+    display: grid;
+    grid-template-columns: minmax(0, 1fr) 40rem;
```

Comment:
don't use media queries, use instead ui-store 
```suggestion
      .host-network-view-mobile {
    display: grid;
    grid-template-columns: minmax(0, 1fr) 40rem;
  }
```